common.mk

```diff
@@ -26,6 +26,98 @@ OPTEE_TEST_OUT_PATH 		?= $(ROOT)/optee_test/out
 
 CCACHE ?= $(shell which ccache) # Don't remove this comment (space is needed)
 
+
+################################################################################
+# Check coherency of compilation mode
+################################################################################
+
+ifneq ($(COMPILE_NS_USER),)
```

Comment:
All good below, but since this is the common.mk file I wonder if we can merge this as it is without affecting the other platforms supported i build.git? (Not a statement, it's a question).